generate/generate.go

```diff
@@ -142,7 +142,14 @@ func New() Generator {
 					Type: "mount",
 				},
 			},
-			Devices: []rspec.Device{},
+			Devices: []rspec.Device{
+				{
+					Path:  "/dev/fuse",
```

Comment:
Here I want to add a default value to the devices, so that you can better achieve devices-remove and other functions.
You mean to give the assignment value is not good, or value `/dev/fuse` is not good?